src/resources/postcss/base/forms/_tabs.pcss

```diff
@@ -68,18 +69,36 @@
 			&:first-child label {
 				@mixin form-control-select-list-item-first;
 
+				body#top & { /* Enfold compatibility */
```

Comment:
let's move all these overrides to the bottom in a new wrapper, like so:

```css
/* Enfold compatibility */
body#top & .tribe-common-form-control-tabs {
    li {
        &:first-child label {
            /* style overrides */
        }
        ...
    }
    ...
}
```

This way we can keep all the override styles in one place rather than having them spread out in the file. In the final CSS file they'll have their own selectors anyway so it won't create a larger CSS file.

Reply:
I did it that way because that's what it said in the doc.

Reply:
You mean the CSS architecture document, right? Yeah, I know, but template overrides are a bit different. Thinking of maintenance, it's going to be quite difficult to maintain style overrides if they're interspersed all over a specific component/element's styles rather than all in one place. And this won't just be for one theme, this is for all themes, for all components.

I'll update the CSS architecture document to reflect this.